lib/sqlalchemy/engine/result.py

```diff
@@ -1709,7 +1709,7 @@ def __init__(self, cursor_metadata, results):
         self._unique_filter_state = results[0]._unique_filter_state
         self._yield_per = results[0]._yield_per
 
-        # going to try someting w/ this in next rev
+        # going to try something w/ this in next rev
```

Comment:
**mike bayer** (zzzeek) wrote:

I wonder what I was going to try? :)

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/2507